components/eamxx/src/physics/cld_fraction/cld_fraction_main_impl.hpp

```diff
@@ -31,11 +33,15 @@ ::main(
     const auto oqi   = ekat::subview(qi,   i);
     const auto oliq_cld_frac = ekat::subview(liq_cld_frac, i);
     const auto oice_cld_frac = ekat::subview(ice_cld_frac, i);
-    const auto otot_cld_frac = ekat::subview(tot_cld_frac,  i);
+    const auto otot_cld_frac = ekat::subview(tot_cld_frac, i);
+    const auto oice_cld_frac_4out = ekat::subview(ice_cld_frac_4out, i);
+    const auto otot_cld_frac_4out = ekat::subview(tot_cld_frac_4out, i);
 
-    calc_icefrac(team,nk,oqi,oice_cld_frac);
+    calc_icefrac(team,nk,1e-14,oqi,oice_cld_frac);
```

Comment:
our threshold used to be 1e-12 before this PR, right? Did you mean to change the threshold here?

Reply:
yes it was, this is a typo, good catch!